setup.py

```diff
@@ -108,27 +108,49 @@ def download_mafft(kind="deb") -> Path:
     return deb_filepath
 
 
+README = Path("README.md").read_text()
+
+install_requires = [
+    "biopython",
+    "numpy",
+    "pandas",
+    "scikit-learn",
+    "weblogo",
+    "Pillow",
+    # :note: pytest should be under tests_require, but this doesn't seem to work
+    "pytest-regtest",
+    "pytest",
+    "multipledispatch",
+    "python-codon-tables",
+]
+
 setup(
     name="seqlike",
-    version="1.1.6",
+    version="1.1.7.dev1",
     packages=find_packages(),  # https://stackoverflow.com/a/22442340
     cmdclass={"install": InstallWrapper},
-    install_requires=[
-        "biopython",
-        "numpy",
-        "pandas",
-        "scikit-learn",
-        "weblogo",
-        "Pillow",
-        # :note: pytest should be under tests_require, but this doesn't seem to work
-        "pytest-regtest",
-        "pytest",
-        "multipledispatch",
-        "python-codon-tables",
-    ],
+    long_description=README,
+    long_description_content_type="text/markdown",
+    url="https://github.com/modernatx/seqlike",
+    project_urls={
+        "Documentation": "https://modernatx.github.io/seqlike",
+        "Source Code": "https://github.com/modernatx/seqlike",
+        "Issue Tracker": "https://github.com/modernatx/seqlike/issues",
+    },
```

Comment:
@ericmjl I also added this metadata. Please let me know, if this looks okay.

```suggestion
    long_description=README,
    long_description_content_type="text/markdown",
    url="https://github.com/modernatx/seqlike",
    project_urls={
        "Documentation": "https://modernatx.github.io/seqlike",
        "Source Code": "https://github.com/modernatx/seqlike",
        "Issue Tracker": "https://github.com/modernatx/seqlike/issues",
    },
```

Reply:
Amazing add, thank you very much!